sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/BeamSqlCliTest.java

```diff
@@ -260,7 +260,7 @@ public void test_time_types() throws Exception {
         "INSERT INTO test_table VALUES ("
             + "DATE '2018-11-01', "
             + "TIME '15:23:59', "
-            + "TIMESTAMP '2018-07-01 21:26:07.123' )");
+            + "TIMESTAMP '2018-07-01 21:26:07' )");
```

Comment:
Why remove the milliseconds from this test case?
If this no longer works then this would imply a backwards compatibility issue to me.

Reply:
I found the cause of the implicit truncation, it is fixed now.